style.css

```diff
@@ -67,11 +75,183 @@ li {
   align-items: center;
   text-align: center;
   color: #6070ff;
+  width: 100%;
+  gap: 70%;
+  background-color: #fff;
+  margin-bottom: -5px;
+  justify-content: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+.middle-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 0 auto;
+  gap: 50%;
   width: 90%;
+  border-radius: 12px;
+  box-sizing: border-box;
+  border: 1px solid #dfe1e6;
+  padding: 14px;
+  margin-top: 30%;
+  background: #fff;
+}
+
+.tonic {
+  align-items: center;
+}
+
+#title {
+  font-size: 1.5em;
+  font-style: bold;
+  font-weight: 900;
+}
+
+.details {
+  font-style: normal;
+  font-size: 1em;
+  display: flex;
+  padding: 0;
   justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0;
+}
+
+.canopy {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1em;
+  color: #344563;
+}
+
+.spec {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1em;
+  color: #7a869a;
+}
+
+.lang {
+  font-style: normal;
+  font-size: 1em;
+  display: flex;
+  padding: 0;
+  justify-content: space-around;
+  width: 80%;
+  color: #6070ff;
+  margin: 0;
+}
+
+.hcj {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding-left: 7px;
+  padding-right: 7px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.hcj p {
+  margin: 0;
+}
+
+.see {
+  margin: 5%;
+}
+
+.button {
+  background-color: #fff;
+  border-color: #6070ff;
+  font-style: normal;
+  font-size: 1em;
+  color: #6070ff;
+  transition-duration: 0.4s;
+}
+
+button:hover {
+  background: #f7f7f9;
+  border-radius: 8px;
+}
+
```

Comment:
- [ ] Kindly add border=radius to your button to match the given template. You are required to stick to design template as much as possible.
### Your design 
![image](https://user-images.githubusercontent.com/94551484/219843235-204d9f66-9d3b-4547-90d4-19fad1947bfc.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/219843268-43bd1036-e37e-4ca5-a683-10e3a700832b.png)

- [ ] Kindly add a hover and pressed state to match the given template. You can use the `:active` pseudo-class for the pressed state.
![image](https://user-images.githubusercontent.com/94551484/219843318-f5d04051-adf1-449d-bda6-c825b024a834.png)